.pre-commit-config.yaml

```diff
@@ -33,7 +33,7 @@ repos:
         entry: golangci-lint run --enable-only goimports --fix
         types: [go]
         language: golang
-        pass_filenames: true
+        pass_filenames: false
```

Comment:
without this I was getting the following errors. This part of the lint is super fast anyway
```
ERRO [linters_context] typechecking error: named files must all be in one directory; have src/pkg/cluster and src/pkg/k8s 
ERRO [linters_context] typechecking error: named files must all be in one directory; have src/pkg/cluster and src/pkg/packager 
ERRO [linters_context] typechecking error: named files must all be in one directory; have src/pkg/cluster and src/config
```